apps/next-evans/pages/[friend]/[friend_slug].tsx

```diff
@@ -157,8 +200,43 @@ const Friend: React.FC<Props> = ({
   description,
   quotes,
   documents,
+  residences,
+  born,
+  died,
 }) => {
   const onlyOneBook = documents.length === 1;
+  const isCompilations = name === `Compilations`;
```

Comment:
i don't think this test will work for spanish... maybe change it to test if the name starts with `Compila`